Sources/XcbeautifyLib/Parser.swift

```diff
@@ -1,246 +1,125 @@
 public class Parser {
-    public init() {}
+    
+    private let colored: Bool
+    
+    private let additionalLines: () -> String?
```

Comment:
Moved parameters to the Parser itself out of the method, which should return beautified string